code/socialDistribution/templates/tagtemplates/author.html

```diff
@@ -0,0 +1,42 @@
+{% load static %}
+<div class="author-tile card shadow border-light bg-white m-5 h-100">
+    <div class="card-body">
+        <div class="row">
+            <div class="col-2 d-flex align-items-center justify-content-center h-100 ml-2">
+                <img src="{% static 'assets/blank-profile-picture.png' %}"
+                    class="profile-img align-middle" alt="">
+            </div>
+            <div class="col">
+                <h5 class="card-title h3">{{author.displayName}}</h5>
+                <h6 class="card-subtitle mb-2 text-muted">{{author.get_host_type_display}}</h6>
+                <div>
+                    {% if not profile_card %}
+                        {% if author.post__count == 1 %}
+                            <p>{{author.post__count}} public post</p>
+                        {% else %}
+                            <p>{{author.post__count}} public posts</p>
+                        {% endif %}
+                    {% else %}
+                        <div class="author-card-opt d-flex ml-10">
+                            {% if friend %}
+                                <p class="text-muted px-1">Friend &#8226;</p>
+                                <a class="px-1" href="{% url 'socialDistribution:unfriend' author.id %}">Un-Friend</a>
+                            {% elif followed %}
+                                <p class="text-muted px-1">Followed &#8226;</p>
+                                <a class="px-1" href="{% url 'socialDistribution:unfollow' author.id %}">Un-Follow</a>
+                            {% elif not author_is_user %}
+                                <a class="px-1" href="{% url 'socialDistribution:befriend' author.id %}">
+                                    {% if accept_friend %}
+                                        Accept friend request
+                                    {% else %}
+                                        Befriend
```

Comment:
Would it make sense to replace "Befriend" with "Follow" to better reflect the initial follow action?

Reply:
I agree, it makes more sense. I'll rename it and change it in the mockups.
Thanks for the review ♥